packages/jsx/src/utils.test.ts

```diff
@@ -0,0 +1,75 @@
+import { atom, Atom, AtomMaybe, CtxSpy, isAtom } from '@reatom/core'
+import { createTestCtx } from '@reatom/testing'
+import * as assert from 'uvu/assert'
+import { cn } from './utils'
+
+describe('reatomClassName', () => {
+  const ctx = createTestCtx()
+
+  it('handles falsy correctly', () => {
+    assert.is(ctx.get(cn(false)), '')
+    assert.is(ctx.get(cn(true)), '')
+    assert.is(ctx.get(cn(null)), '')
+    assert.is(ctx.get(cn(undefined)), '')
+    assert.is(ctx.get(cn(atom(undefined))), '')
+    assert.is(ctx.get(cn(() => undefined)), '')
+  })
+
+  it('handles falsy object correctly', () => {
+    assert.is(ctx.get(cn({
+      a: '',
+      b: 0,
+      c: NaN,
+      d: false,
+      e: null,
+      f: undefined,
+      g: atom(undefined),
+    })), '')
+  })
+
+  it('handles falsy array correctly', () => {
+    assert.is(ctx.get(cn([
+      '',
+      null,
+      undefined,
+      {},
+      [],
```

Comment:
It's a bit inconsistent, that we treat `{}` and `[]`   
In js world they truly, and in test above ('handles falsy object correctly') we not test it ether